15.CSS_Form/assets/scss/style.scss

```diff
@@ -0,0 +1,111 @@
+.account {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__container {
+    min-width: 570px;
+    min-height: 590px;
+    background-color: $color_white-100;
+    border: 1px solid $color_grey-100;
+    border-radius: 6px;
+    padding: 35px 35px 15px;
+  }
+
+  &__form {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 12px;
+
+    &-inputbox,
```

Comment:
&-inputbox as for my it's bad practice to use such class names you will have issues with huge class names like this 
.locl-campaign-planner-item-activity